cmd/validator/credentials/set/process.go

```diff
@@ -238,10 +252,14 @@ func (c *command) generateOperationsFromMnemonic(ctx context.Context) error {
 	maxDistance := 1024
 	// Start scanning the validator keys.
 	lastFoundIndex := 0
+	found_validator_count := 0
 	for i := 0; ; i++ {
+		// if no validators have been found in the last 1024 indices, stop scanning
```

Comment:
Comments need to start with capital letters and end with a period, or they will fail linting.